docs/source/installation.rst

```diff
@@ -44,17 +44,23 @@ To install ``sktime`` with maximum dependencies, including soft dependencies, in
 
     pip install sktime[all_extras]
 
+``sktime`` also comes with dependency sets specific to learning task, i.e., estimator scitype.
+These are curated selections of the most common soft dependencies for the respective learning task.
+The available dependency sets are of the same names as the respective modules:
+``forecasting``, ``transformations``, ``classification``, ``regression``, ``clustering``, ``param_est``,
+``networks``, ``annotation``, ``alignment``.
+
```

Comment:
I thought about that, but then I realized it would be a long list and take up much space (and attention) in a section that should be very short. No strong opinion though, what is your preference given this observation?

Reply:
Since theae are important from user point of view, i.e. they start interacting with `sktime` through these, I feel it's worth highlighting them (and possibly as a table along with contents and descriptions in future, e.g. the way `pandas` does it). Users may go through these and decide what all (or their subsets based on use cases) do they need.

I don't have a super strong opinion,  but if you also don't have one, I'll keep my suggestion to switch to bullets :)

Reply:
ok